Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_expeditions_aberrant_flesh.yml

```diff
@@ -357,17 +367,68 @@
     speedModifierThresholds:
       120: 0.7
       160: 0.5
-  - type: MovementSpeedModifier
-    baseWalkSpeed: 3.5
-    baseSprintSpeed: 4
   - type: MeleeWeapon
+    autoAttack: true
+    attackRate: 1.5
     soundHit:
         path: /Audio/Weapons/Xeno/alien_claw_flesh3.ogg
     angle: 0
     animation: WeaponArcClaw
     damage:
       types:
         Slash: 12
+        Structural: 30
+  # Ghost role stuff
+  - type: GhostRole
+    allowMovement: true
+    name: ghost-role-information-aberrant-flesh-horror-dungeon-name
+    description: ghost-role-information-aberrant-flesh-horror-dungeon-description
+    rules: ghost-role-information-aberrant-flesh-horror-dungeon-rules
+    raffle:
+      settings: default
+  - type: GhostTakeoverAvailable
+  - type: ActionGun
+    action: ActionBoneSpike
+    gunProto: BoneSpikeGun
+
+- type: entity
+  id: BoneSpikeGun
+  name: bone spike
+  description: Shoots short salvo of bone spikes.
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: RechargeBasicEntityAmmo
+    rechargeCooldown: 5
+    rechargeSound:
+      path: /Audio/Animals/space_dragon_roar.ogg
+  - type: BasicEntityAmmoProvider
+    proto: DartAberrantFleshHorrorSpikeSpread #DartAberrantFleshHorrorSpike
+    capacity: 3
```

Comment:
Having three of these cached at once seems a bit gamey to me, but it also seems fairly weak at range.  Not sure how it works out in practice, but I like the ability.

Looking forward to getting that feature that gives the actions the little status by the mouse curser from upstream.

Reply:
It takes 6 shots point blank (to ensure that all spikes land) to crit unarmoured Urist, but this ability is less about damage - and more about slowing down the target with gastrotoxins: you probably want to land at least 2 good shots (4-6 spikes) to have a decent chance for the effect to trigger.

Regardless, reduced cache to two.